### DISCUSSION TITLE
Livewire not work in pair Alpine3 + Vue2

### DISCUSSION THREAD

**Discussion by rik43** [livewire/livewire#3653] [7. Bugs (archived)]
i use 
vue@2.5.17
livewire@2.5.5
livewire-vue@latest
alpine@3.2.4

Result: alpine works fine, LIVEWIRE doesn't work (no attached js event handlers on elements, no http requests).

with alpine@2.8.2 everything works well.

with alpine@3.2.4 only works if vue2 is disabled.

what could be the problem?

**Comment by rik43** [comment]
i`m migrate to vue3 and... it still doesn't work.

the good thing is that I can turn off vue completely on the pages with livewire. 

although this is not the best solution.

**Comment by joshhanley** [comment]
@rik43 do you get any console errors? If so what? Can you screenshot them?

**Reply by rik43** [reply]
no errors.

**Reply by joshhanley** [reply]
@rik43 hmm ok strange. Well one thing, is Vue3 isn't supported by the Livewire Vue plugin, see here https://github.com/livewire/vue/issues/33

I'm not sure I can be of much help as I personally don't use Vue with Livewire.

But if you can setup an example repo that demonstrates the issue, I'd be happy to pull it down and have a look at it to see if I can work out what is going on. Please make sure it's the latest Laravel/Livewire/Alpine but use Vue2 not Vue3.

Hope this helps!

**Reply by rik43** [reply]
Thanks. i will do the repo when the time comes.

use Vue with Livewire is necessary for legacy projects for a soft migrate.

**Reply by joshhanley** [reply]
@rik43 that'd be great thanks!

Yep agreed 🙂
